lib/ReactViews/SidePanel/SidePanel.jsx

```diff
@@ -44,7 +44,14 @@ function EmptyWorkbench(props) {
         </Text>
       </Box>
       <Spacing bottom={10} />
-      <Box column paddedRatio={3}>
+      <Box
+        column
+        paddedRatio={3}
+        positionAbsolute
```

Comment:
You'll need some responsive-to-height styles for this as you can end up with this case
![image](https://user-images.githubusercontent.com/9134432/85814897-68469180-b7aa-11ea-8a1d-77fc2fd37873.png)

Reply:
Hmm should the helpful hints just be turned off at that point? It's either going to get clipped to like the first line or not even be visible

Reply:
Could just make it overflowy: scroll like it does when workbench has long list of items

Reply:
overflow-y:auto rather

Reply:
Ok so I made the "Your workbench is empty" text responsive as well!

Reply:
It didn't look very nice with the scroll bar 😬

Reply:
I just realised I only debugged when I had inspect open, so it still didn't fit well if I zoom in to 150% with the entire screen........... I think I might have to resort to overflow-y as a quick fix and let vic do the design for tiny screens